views/partials/project/gallery.blade.php

```diff
@@ -0,0 +1,5 @@
+@gallery([
+    'classList' => ['project-gallery'],
+    'list' => $list,
```

Comment:
Finally replace `$list` with namespaced property eg. `$project['gallery` or `$project['gallery]['list']` (else the !empty($data['gallery']) will still show the view even though $list may be undefined)